srcpkgs/serviio/template

```diff
@@ -0,0 +1,61 @@
+# Template file for 'serviio'
+pkgname=serviio
+version=2.1
+revision=1
+archs="x86_64"
+depends="openjdk11 ffmpeg"
+short_desc="Free DLNA media server"
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="custom"
+homepage="http://www.serviio.org/"
+distfiles="http://download.serviio.org/releases/${pkgname}-${version}-linux.tar.gz"
+checksum=51748160b38e31ba54db1b11381d3d9d705852ed7757ad111e881327a16fb757
+
+do_install() {
+
+    #Define variable for Serviio home directory
+    SERVIIO_HOME=/usr/share/java/$pkgname
```

Comment:
This path shouldn't have a leading slash., if it's going to be used for `vmkdir` statements.